backend/app/services/knowledge/knowledge_service.py

```
@@ -350,12 +352,189 @@ def get_knowledge_base(
+        """Apply read/edit direct-access requirements to a Kind query."""
+        from app.models.resource_member import MemberStatus, ResourceMember
+        from app.models.share_link import ResourceType
+
+        user = KnowledgeService._get_user_or_raise(db, user_id)
+        accessible_groups = get_user_groups(db, user_id)
+        organization_names = {
+            row[0]
+            for row in db.query(Namespace.name)
+            .filter(
+                Namespace.level == GroupLevel.organization.value,
+                Namespace.is_active == True,
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)